.swcrc

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://json.schemastore.org/swcrc",
   "jsc": {
-    "target": "es2020",
+    // "target": "es2020",
```

Comment:
build gives an error `env` and `jsc.target` cannot be used together.

deferred to env over target, but that was just a guess